oat-sa/extension-tao-sync
# [TAO-8705] [FE]DEPP - 19EVA2-19EVA6 - user interface for exporting a sync file when syncronisation fails

Related to : https://oat-sa.atlassian.net/browse/TAO-8705
 
When an error occurs during synchronization process, display export button to allow user to download synchronization data and import it later

Note:
Import UI is out of the scope for this PR  

#### How to test
 
- install test center and offline instances of `DEPP` extension using composer and seed files from https://oat-sa.atlassian.net/browse/TAO-8681
- create a test center with an assigned sync manager and some delivery
- login into offline instance as a sync manager
- try to run synchronization
- when synchronization fails, click export button
- processing screen should appear
- zip with sync data should be downloaded at the end of the preparation data process
- the initial sync screen should be shown
  
#### Dependencies
 
Requires :
 - [x] https://github.com/oat-sa/extension-tao-sync/pull/153
